src/tests/integration/steps/posting-feedback.ts

```diff
@@ -3,10 +3,11 @@ import * as fetchMock from 'fetch-mock'
 import { Mocks } from '../mocks'
 import { ensureActiveTab, ensureActiveFeedbackTarget } from '../../../selectors'
 
-export function postingFeedback(mocks: Mocks): void {
-  describe(`post feedback`, () => {
+export function postingFeedback(mocks: Mocks, opts: { result: 'success' | 'unauthorized' }): void {
```

Comment:
Posting feedback can also fail due to problems with Twitter, problems with the server or bad connection. Can you please add a test to make sure that we stay authenticated in this case?

Reply:
added